.circleci/config.yml

```diff
@@ -92,6 +92,11 @@ commands:
     steps:
       - run: echo 'CIRCLE_WORKING_DIRECTORY="${CIRCLE_WORKING_DIRECTORY/#\~/$HOME}"' >> $BASH_ENV
 
+  add-tools-scripts-to-path:
```

Comment:
Just name the script by path when it's used, as we already do for other scripts in that directory.